heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesScheduler.java

```diff
@@ -400,9 +404,9 @@ protected ArrayNode getPorts(ObjectMapper mapper) {
   /**
    * Get the command that will be used to retrieve the topology JAR
    */
-  protected String getFetchCommand() {
-    return "cd /opt/heron/ && curl " + Runtime.topologyPackageUri(runtime).toString()
-            + " | tar xvz";
```

Comment:
the downloader extracts the tar.gz on the file. There is a test for this (ExtractorTests under heron/downloaders/tests)